docs/tracing/create-tracing-app.md

```diff
@@ -0,0 +1,254 @@
+- Traces Token: For sending trace data to Sematext Cloud
+- Service Configuration: All configured services ready to send traces
+
+### Ready-to-Use Features
+
+- Tracing Overview: Key metrics and service health
+- Traces Explorer: Search and filter traces
+- Individual Trace Analysis: Detailed span inspection
+- Alert Configuration: Ready to set up performance alerts
+
+## SDK Examples
+
+Here are quick examples for popular languages:
+
+### Java
+```bash
+# Download OpenTelemetry Java agent
+wget https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/latest/download/opentelemetry-javaagent.jar
+
+# Run with automatic instrumentation
+java -javaagent:opentelemetry-javaagent.jar \
+  -Dotel.service.name=your-service-name \
+  -Dotel.exporter.otlp.endpoint=http://localhost:4338 \
+  -Dotel.exporter.otlp.protocol=http/protobuf \
+  -jar your-application.jar
+```
+
+### Python
+```bash
+# Install OpenTelemetry packages
+pip install opentelemetry-distro[otlp]
+opentelemetry-bootstrap -a install
+
+# Configure and run
+export OTEL_SERVICE_NAME=your-service-name
+export OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:4338
+export OTEL_EXPORTER_OTLP_PROTOCOL=http/protobuf
+
+opentelemetry-instrument python your_app.py
+```
+
+### Node.js
+```bash
+# Install dependencies
+npm install @opentelemetry/auto-instrumentations-node @opentelemetry/exporter-trace-otlp-http @opentelemetry/sdk-node
+
+# Create tracing.js and run
+node -r ./tracing.js your-app.js
+```
+
+For complete setup instructions, the wizard provides detailed, language-specific guidance.
+
+## Best Practices
+
+### Naming Conventions
+
+- Apps: Use environment prefixes (`prod-api`, `staging-web`)
+- Services: Use descriptive names (`user-authentication`, `payment-processor`)
+- Keep names concise but clear
+
+### Planning Your Setup
+
+- One App per Service: Separate Tracing Apps for different microservices
+- Environment Separation: Different Apps for prod/staging/dev
+- Infrastructure Apps: Reuse for related services on same infrastructure
+
+## Troubleshooting
+
+### No Traces Appearing
+
+- Verify agent is running and shows "Host Detected"
+- Check OTLP endpoint configuration in your application
+- Ensure service names match between App configuration and code
+- Review the [Troubleshooting Guide](/docs/tracing/troubleshooting/)
+
+### Agent Issues
+
+- Confirm Infra App token is correct
+- Check network connectivity between agent and application
+- Verify ports 4337/4338 are accessible
+- Review agent logs for configuration errors
+
+## Next Steps
+
+After completing App creation:
+
+1. Explore Traces: Navigate to [Traces Explorer](/docs/tracing/reports/explorer/) to see your traces
+2. Set Up Alerts: Create [performance alerts](/docs/tracing/alerts/creating-alerts/) for key metrics  
+3. Add Custom Instrumentation: Learn about [manual instrumentation in SDKs](/docs/tracing/sdks/)
+4. Optimize Performance: Configure [sampling strategies](/docs/tracing/sampling/)
```

Comment:
Might be handy to have a page called "Cost Optimizations" and such -- we have something like that for Logs -- so we can easily point people to that.

Reply:
Added cost optimization page.